### PR TITLE
VACMS-10099: Place Q&As Modal in the CMS is not available with keyboard

### LINKED ISSUE #10099: Place Q&As Modal in the CMS is not available with keyboard
## Describe the defect
On FAQ page content type (and soon the Resources and Support Detail page) the editor has the ability to add a Q&A Group with multiple Q&As.  If the user is tabbing through the form and activates the button for "Place Q&As" a modal opens that should allow the editor to choose which questions they would like to display in the group.  However, the keyboard focus is taken to the close button of the modal and the user is unable to tab to anything else in the modal, focus is stuck to the close button

## To Reproduce
1. Go to an FAQ page within the CMS. [Example page](https://staging.cms.va.gov/node/8233/edit)
2. Scroll down to Q&A Groups
3. Use the keyboard to tab to the "Place Q&As" button
4. Hit enter to activate the button and activate the modal
6. Hit tab to move through the modal
7. Notice that the green focus indicator is on the X to close the modal
8. Continue trying to tab through the modal and notice that the focus does not move from the X Close button

## AC / Expected behavior

- [x] When the modal opens, the user should be able to tab through all focusable elements within the modal
- [x] The user should not be able to tab to anything outside of/behind the modal
- [x] The modal should be able to be closed with the Close button and/or the ESC keyboard key.  Also in this case the "Select Q&As" button as that follows the use-case workflow
- [x] When the modal is closed (using any of the above scenarios), the focus should be taken back to the original trigger, in this case the "Place Q&As" button

## Screenshots
Place Q&As button
image

Modal with focus indicator on close button
image

 ## Additional context
This is also happening on the Select Benefits Hubs modal further down in the same content type. I do not know if one solution would resolve both instances or if I should create a separate ticket for the Benefits Hubs modal.  

## Desktop (please complete the following information if relevant, or delete)
Issue is occurring on Chrome, FF and Edge

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Closes #10099.

## Testing done
Tested on my local.

## QA steps

1. Go to an FAQ page within the CMS. 
    https://staging.cms.va.gov/node/8233/edit
2. Scroll down to Q&A Groups
3. Use the keyboard to tab to the "Place Q&As" button
4. Hit enter to activate the button and activate the modal
5. Hit tab to move through the modal
6. The focus indicator should move back to the X
7. Once the X int he top right corner of the modal is back in focus hit neter to exit the modal.

### Definition of Done

- [x] Acceptance Criteria in related issue are met.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -376,7 +376,8 @@
                 "Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2021-05-05/3111456-34.patch"
             },
             "drupal/entity_browser": {
-                "Entity browser cardinality validation": "https://www.drupal.org/files/issues/0001-Issue-2856138-by-gordon-Error-messages-are-not-being.patch"
+                "Entity browser cardinality validation": "https://www.drupal.org/files/issues/0001-Issue-2856138-by-gordon-Error-messages-are-not-being.patch",
+                "Make modal iframe tab accessible": "https://www.drupal.org/files/issues/2021-01-07/3191302-3.patch"
```
Comment (ndouglas) [inline composer.json:380]: Needs a `composer update --lock` since `composer.json` was updated.

**Review by ndouglas — DISMISSED** [review]
QA steps followed, modal behaved as described 😁

**Comment by ndouglas** [comment]
D'oh, sorry @edmund-dunn, I shouldn't've merged my PR until this one was merged 🤦🏻‍♂️

**Comment by ndouglas** [comment]
This was approved and the only change I made was to resolve a merge conflict that arose, so I'm forcibly merging this.

**Comment by ndouglas** [comment]
Er, just kidding.  I'll wait for @laflannery's review.  Man, I'm jumping the gun everywhere today.

**Review by ndouglas — APPROVED** [review]

**Review by laflannery — APPROVED** [review]

**Pull request merged**